app/models/edition.rb

```diff
@@ -279,8 +277,12 @@ def search_format_types
     [Edition.search_format_type]
   end
 
+  def self.publicly_visible_and_available_in_english
+    with_translations(:en).where(state: %w(published withdrawn))
```

Comment:
It would be better to use the scope `.publicly_visible` which in turn uses the constant.